scripts/release/publish-docs-content.sh

```diff
@@ -52,17 +53,20 @@ do
   fi
 done
 
-# Move guide files over to $repoPath/guides
+# Copy guide files over to $repoPath/guides
 for filename in $overviewFiles*
 do
   if [ -f $filename ]; then
     cp -r $filename $repoPath/guides
   fi
 done
 
-# Move highlighted examples into $repoPath
+# Copy highlighted examples into $repoPath
 cp -r $examplesSource/* $repoPath/examples
 
+# Copy example plunker assets
+rsync -a $docsPath/plunker $repoPath/plunker
```

Comment:
I'm not sure `rsync` is supported in the Git Bash. But as we run this on the CI it's fine. Just think you can do the same with `cp`.